packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -652,19 +653,28 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     // If autoComplete is on, attempt to find a match from the available options
     if (this.props.autoComplete === 'on') {
       // If autoComplete is on, attempt to find a match where the text of an option starts with the updated value
-      const items = currentOptions
+      let items = currentOptions
         .map((item, index) => {
           return { ...item, index };
         })
         .filter(
           option => option.itemType !== SelectableOptionMenuItemType.Header && option.itemType !== SelectableOptionMenuItemType.Divider
-        )
-        .filter(
+        );
+
+      if (this.props.autoCompleteFullString === 'on') {
+        items = items.filter(option =>
+          this._getPreviewText(option)
+            .toLocaleLowerCase()
```

Comment:
This behavior (line 607) should be part of your unit test

Reply:
Done.